ca/config.go

```diff
@@ -274,6 +274,9 @@ func LoadSecurityConfig(ctx context.Context, rootCA RootCA, krw *KeyReadWriter)
 
 	// Check to see if this certificate was signed by our CA, and isn't expired
 	if _, err := X509Cert.Verify(opts); err != nil {
+		fmtString := "Jan 02 15:04:05 2006 MST"
+		log.G(ctx).WithError(err).Debugf("invalid certificate issued by %s (not before %s, not after %s)",
```

Comment:
I'd think this should be an error or warning log rather than a debug log. This information should be made very visible.

Ideally, we should include most useful information in the error that gets bubbled up to the user, so checking the logs is a last resort. Including `NotBefore` / `NotAfter` and the current system time in the user-facing error when time seems like a factor may be a good idea here.